WORKSPACE

```diff
@@ -0,0 +1,15 @@
+# 
+# Copyright 2020 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+    # https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
```

Comment:
I see the "No newline at end of file" warning. Could you fix it (here and everywhere else it appears)?

Reply:
Sure!

Reply:
Still seeing it showing up.

Reply:
Bumping this up again, since I still see the "No newline at end of file" warning. Not sure what's going on here, since this seems to be a recurring issue. If the editor automatically removes this, could you perhaps figure out how to change the settings of the editor?